openpype/hosts/traypublisher/addon.py

```diff
@@ -53,3 +54,64 @@ def launch():
     from openpype.tools import traypublisher
 
     traypublisher.main()
+
+
+@cli_main.command()
+@click.option(
+    "--csv-filepath",
+    help="Full path to CSV file with data",
+    type=click.Path(exists=True),
+    required=True
+)
+@click.option(
+    "--project-name",
+    help="Project name in which the context will be used",
+    type=str,
+    required=True
+)
+@click.option(
+    "--asset-name",
+    help="Asset name in which the context will be used",
+    type=str,
+    required=True
+)
+@click.option(
+    "--task-name",
+    help="Task name under Asset in which the context will be used",
+    type=str,
+    required=False
+)
+@click.option(
+    "--ignore-validators",
+    help="Option to ignore validators",
+    type=bool,
+    is_flag=True,
+    required=False
+)
+def ingestcsv(
+    csv_filepath,
+    project_name,
+    asset_name,
+    task_name,
+    ignore_validators
+):
+    """Ingest CSV file into project.
+
+    This command will ingest CSV file into project. CSV file must be in
+    specific format. See documentation for more information.
+    """
+    if AYON_SERVER_ENABLED:
+        raise RuntimeError(
+            "This feature is not supported in Ayon yet. "
+            "Contact support for more info."
+        )
```

Comment:
As far as I know this alone would make this PR invalid for merging.

[This](https://community.ynput.io/t/openpype-end-of-life-timeline/877/7?u=bigroy) mentions there currently being a requirement for PRs to be AYON compatible.

Reply:
No it doesnt since the description of this PR is stating its future integration. This doen't need to be part of this PR since it is already quite large.